ts_src/index.ts

```diff
@@ -26,8 +26,7 @@ function base (ALPHABET: string): base.BaseConverter {
   const iFACTOR = Math.log(256) / Math.log(BASE) // log(256) / log(BASE), rounded up
 
   function encode (source: Uint8Array | number[]): string {
-    if (source instanceof Uint8Array) {
-    } else if (ArrayBuffer.isView(source)) {
+    if (ArrayBuffer.isView(source)) {
```

Comment:
I agree that this was originally poorly written, but this change makes the code different.

What if source is BOTH `Uint8Array` AND one of the other `else if`?

If you are 100% confident that these conditions will never overlap in any JS runtime ever, then maybe this is OK. But I would rather just leave it as is... or at least re-write it with the same semantic meaning... like wrapping the whole if else chain with a negation of the empty if.

Reply:
Actually, I'm wondering if I should remove the first if statement. The function just accepts `Uint8Array | number[]` clearly.

Reply:
Reverted back to the original code